package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "a11y": "metal-a11y --packages ./packages -c demos/index.html",
+    "a11y": "metal-a11y --packages ./packages -c demos/a11y.html",
```

Comment:
hey @carloslancha, why two files in the `index` and `a11y` demos?

Reply:
`index` is for demo purposes, `ally` only fotos accessiblity. There're times when we are not following accessiblity due to the config we pass to the components (like in ClayList, where we're passing the same items to all instances in behalf of simplicity) but that doesn't mean component itself is not following a11y. That's where `ally.html` shows up.

Reply:
This is good. The bad thing is that we will have another file for maintenance.

Reply:
When we have all Clay+claycss.com+components fully linked in the project demos probably will be useless. Since then...